aemedge/blocks/columns/columns.css

```diff
@@ -530,6 +531,12 @@
   }
 
   @media (width >=992px) {
+    .columns {
+      line-height: 30px;
```

Comment:
Remove the line-height: 30px; instead, please edit lines 254 and 255 of styles.css to the values are "em" instead of "rem" for the size-m. (the rest of the sizes are correct in "rem", I didn't realize Sling had a different one for this).
Should be this:
```
  &.size-m {
    font-size: 1.125em;
    line-height: 1.125em;
    font-weight: 400;
  }
```